test/metabase/lib/drill_thru/zoom_in_timeseries_test.cljc

```diff
@@ -164,3 +176,37 @@
                                               [:field {:temporal-unit :year} (meta/id :orders :created-at)]
                                               "2022-12-01T00:00:00+02:00"]]}]}
                     (lib/drill-thru query -1 drill)))))))))
+
+(def ^:private unit-pairs
+  (let [units @#'lib.drill-thru.zoom-in-timeseries/valid-current-units]
+    (map vector units (rest units))))
```

Comment:
I think the standard way to write this is
```suggestion
    (partition 2 1 units)))
```
It doesn't produce vectors, but that seems to be good enough here.

Reply:
That works for me!